cmd/collector/app/builder/span_handler_builder.go

```diff
@@ -165,8 +165,3 @@ func (c *cassandraSpanHandlerBuilder) getSession() (cassandra.Session, error) {
 	}
 	return c.session, nil
 }
-
-func fixConfiguration(config cascfg.Configuration) cascfg.Configuration {
-	config.ProtoVersion = 4
```

Comment:
@badiib what was the reason for this override? The configuration supports this from yaml, and we're setting it in our internal config files.